software/edge/src/aeris_mesh_agent/aeris_mesh_agent/store_forward_core.py

```
@@ -120,31 +203,80 @@ def sync_connectivity_and_flush(self, publish_callback: PublishCallback) -> bool
+                            record.message_kind,
+                            record.payload,
+                            ReplayMetadata(
+                                delivery_mode="replayed",
+                                original_event_ts=record.event_ts,
+                                replayed_at_ts=time.time(),
+                                dedupe_key=record.dedupe_key,
+                                payload_hash=record.payload_hash,
+                                priority_class=record.priority_class,
+                            ),
+                        )
+                    except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)